### ISSUE TITLE
`machine` ` runner` does not work on aws with our regions

### ISSUE THREAD

**Issue by DavidGOrtega** [iterative/terraform-provider-iterative#335]
After #323 those resource do not work if we use our synthetic regions.
Its because the availabilityZone should be calculated with the region mapping

```go
region := GetRegion(d.Get("region").(string))
availabilityZone := GetAvailabilityZone(region)
```

**Comment by casperdcl** [comment]
Still [failing](https://gitlab.com/iterative.ai/cml/-/pipelines/431735606)?

**Comment by DavidGOrtega** [comment]
> Still [failing](https://gitlab.com/iterative.ai/cml/-/pipelines/431735606)?

[It works](https://gitlab.com/iterative.ai/cml/-/pipelines/431735606), probably the tpi was not updated in the registry yet

**Comment by DavidGOrtega** [comment]
closed via #336 
(for some reason it was not automatically closed)

**Comment by casperdcl** [comment]
> closed via #336 (for some reason it was not automatically closed)

I had manually reopened thinking it wasn't solved yet :)
